src/snippets/scripting/index.md

```diff
@@ -33,4 +33,65 @@
 
 ```typ
 #",this, is a a a a; a. test? string!".matches(regex("(\b[\P{Punct}\s]+\b|\p{Punct})")).map(x => x.captures).join()
-```
\ No newline at end of file
+```
+
+## Create selector matching any values in an array
+
+This snippet creates a selector (that is then used in a show rule) that
+matches any of the values inside the array. Here, it is used to highlight
+a few raw lines, but it can be adapted to any kind of selector obviously
+
+````typ
+#let lines = (2, 3, 5)
```

Comment:
This is your snippet, right? I think adding yourself as an author would be nice. :)

```suggestion
// author: Blokyk
#let lines = (2, 3, 5)
```